src/services/TestResultsService.ts

```diff
@@ -62,16 +63,22 @@ import {
   testTypesSchemaSpecTestGroup1, testTypesSchemaSpecTestGroup2,
   testTypesSchemaSpecTestGroup3, testTypesSchemaSpecTestGroup4, testTypesSchemaSpecTestGroup5
 } from "../models/test-types/testTypesSchemaSpecialistTestsPut";
+import { Injector } from "../models/injector/Injector";
+import { TestDataProvider } from "../handlers/expiry/providers/TestDataProvider";
+import { TestTypeForExpiry } from "../models/TestTypeforExpiry";
```

Comment:
Could doing the same for the models or export * from an index.ts in this folder so we have an isolated module.
Up to you but it could shorten the imports even more